_layouts/default.html

```diff
@@ -17,10 +17,12 @@
       {% assign grid = "grid-content" %}
     {% endif %}
 
-    <main id="main" class="{{ grid }} flow my-content">
+    <main id="main" class="{{ grid }} flow {% unless page.isFrontPage %}my-content{% endunless %}">
```

Comment:
Exclude the base spacer class for the homepage since we have that full bleed pattern